lmn/urls.py

```diff
@@ -21,6 +21,8 @@
     path('notes/detail/<int:note_pk>/', views_notes.note_detail, name='note_detail'),
     path('notes/for_show/<int:show_pk>/', views_notes.notes_for_show, name='notes_for_show'),
     path('notes/add/<int:show_pk>/', views_notes.new_note, name='new_note'),
+    path('notes/<int:note_pk>/delete', views_notes.delete_note, name='delete_note'), #issue 5 delete note by chris
```

Comment:
As noted in the last pull request review, don't leave comments like this. Git tags every line of code you write with your name, and you should linking it to the issue using this pull request.

Reply:
Removed the comment and linked pull request to the issue #5